src/wallet/transaction.h

```diff
@@ -258,6 +258,9 @@ class CWalletTx
     // BlockConflicted.
     std::set<Txid> mempool_conflicts;
 
+    // Set of v3 transactions that spend from this tx
```

Comment:
Comment seems inaccurate: it mentions a set but this only allows for 1 transaction. Should also mention (1) that this is used to stop us from creating another unconfirmed child and (2) this is specifically the in mempool-sibling, as there can be multiple siblings but only 1 in mempool (unless there was a reorg).

Reply:
Done